### PR TITLE
Todo-11-FE: Add BE pagination

### PR DESCRIPTION
**Pull request by itc02**
image
image
image

### CODE REVIEW

File: src/components/mainTable/MainTable.tsx [hunk 1]
```diff
@@ -8,34 +8,66 @@ import TableHead from '@material-ui/core/TableHead';
...
 export const MainTable:React.FC = () => {
-  const todos = useTodos();
   const [ openAddDialog, setOpenAddDialog ] = useState(false);
 
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = useState(pagination.rowsOnPage[0]);
+  const [ todos, setTodos ] = useState([]);
+  const [ currentPer, setPer ] = useState(5);
+  const [ currentPage, setPage ] = useState(1);
+  const [ allTodosCount, setAllTodosCount ] = useState(0);
```
Comment (arpinehp) [inline src/components/mainTable/MainTable.tsx]: what does it mean? do you need the count of all records from DB?

Comment (arpinehp) [reply src/components/mainTable/MainTable.tsx]: if yes, then you should return the `total_record_count` form BE

File: src/components/mainTable/MainTable.tsx [hunk 2]
```diff
@@ -8,34 +8,66 @@ import TableHead from '@material-ui/core/TableHead';
...
+  }
+
+  const pagesCount = () => {
+    return todos.length === 0 ? 1 : Math.ceil(allTodosCount / currentPer);
+  }
+
+  useEffect(() => {
+    getTodos(currentPer, currentPage);
+    axios.get(`${routes.server}/${routes.todos}/count`).then(res => {
+      setAllTodosCount(parseInt(res.data.count));
```
Comment (arpinehp) [inline src/components/mainTable/MainTable.tsx]: :facepalm:

Comment (itc02) [reply src/components/mainTable/MainTable.tsx:66]: maybe I can use `todos.length`?

**Pull request merged**